examples/cms-makeswift/README.md

```diff
@@ -0,0 +1,74 @@
+# A statically generated landing page with Next.js and Makeswift
+
+This example showcases how you can use [Makeswift](https://www.makeswift.com/) to visually build statically generated pages in Next.js.
+
+## Deploy your own
+
+Deploy the example using [Vercel](https://vercel.com?utm_source=github&utm_medium=readme&utm_campaign=next-example):
+
+[![Deploy with Vercel](https://vercel.com/button)](https://vercel.com/new/git/external?repository-url=https://github.com/vercel/next.js/tree/canary/examples/DIRECTORY_NAME&project-name=cms-makeswift&repository-name=cms-makeswift)
+
+## Demo
+
+### [https://nextjs-makeswift-example.vercel.app/](https://nextjs-makeswift-example.vercel.app/)
```

Comment:
See also: https://github.com/vercel/next.js/pull/38018/files#r907833127

Up to you where to point, but at least the README and the link in the docs should probably link to the same example. :+1: